src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.css

```diff
@@ -20,3 +20,26 @@
   margin-top: 0.5em;
   font-size: small;
 }
+
+/*Style for the search box in the tree-node accordion on the left side*/
+:host >>> .gb-tree-filter-input > input {
```

Comment:
same here, duplicated in gb-tree-search.component.css?

Reply:
ideally we want to confine the css the its component, in this case, the styles for gb-tree-search component will be in the gb-tree-search.css